skyjo.py

```diff
@@ -613,21 +752,50 @@ def actions(skyjo: Skyjo) -> np.ndarray[tuple[int], np.uint8]:
         for i in range(FINGER_COUNT):
             row, column = divmod(i, COLUMN_COUNT)
             if table[0, row, column, FINGER_HIDDEN]:
-                mask[MASK_FLIP + i] = 1  # You both flip or replace
+                mask[MASK_FLIP + i] = 1
```

Comment:
we can probably just inline these now

```suggestion
                mask[MASK_FLIP + I] = mask[MASK_REPLACE + i] = 1
```